lib/cache.js

```diff
@@ -70,6 +71,22 @@ module.exports = class Cache {
     return content
   }
 
+  cleanDownloadUrl(url) {
+    return url
+      .replace(
+        `https://github.com/${this.config.account}/${
+          this.config.repository
+        }/releases/download/v`,
+        `${this.config.proxyUrl}/`
+      )
+      .replace(
+        `https://github.com/${this.config.account}/${
+          this.config.repository
+        }/releases/download`,
+        this.config.proxyUrl
+      )
```

Comment:
Why are you doing it like this? Two chained replaces? Are you just returning an entirely new string?